tests/common.go

```diff
@@ -4854,10 +4869,10 @@ func updatePxRuntimeOpts() error {
 }
 
 //StartTorpedoTest starts the logging for torpedo test
-func StartTorpedoTest(testName, testDescription string, tags []string) {
+func StartTorpedoTest(testName, testDescription string, tags map[string]string, testRepoID int) {
 	TestLogger = CreateLogger(fmt.Sprintf("%s.log", testName))
 	SetTorpedoFileOutput(log, TestLogger)
-	dash.TestCaseBegin(testName, testDescription, "", tags)
+	dash.TestCaseBegin(testName, testDescription, strconv.Itoa(testRepoID), tags)
```

Comment:
why do we need to make the conversion here, maybe we can pass that variable as a string?

Reply:
testRailID is declared and used as int all our the place, which is correct. But Aetos API expects this to be a string. So making the conversion.